x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/action/TransportMlMemoryAction.java

```diff
@@ -106,7 +107,11 @@ protected void masterOperation(
                 .setTimeout(request.ackTimeout())
                 .execute(delegate.delegateFailureAndWrap((delegate2, nodesStatsResponse) -> {
                     TrainedModelCacheInfoAction.Request trainedModelCacheInfoRequest = new TrainedModelCacheInfoAction.Request(
```

Comment:
Its really weird that the action itself just doesn't have a min_transport version :/